.github/workflows/build-apps.yml

```diff
@@ -215,7 +215,6 @@ jobs:
           CSC_LINK: ${{ secrets.APPLE_CERTIFICATE }}
           CSC_KEY_PASSWORD: ${{ secrets.APPLE_CERTIFICATE_PASSWORD }}
           CSC_KEYCHAIN: ${{ secrets.APPLE_SIGNING_IDENTITY }}
-          WINDOWS_CERTIFICATE_THUMBPRINT: ${{ secrets.WINDOWS_CERTIFICATE_ZOO_THUMBPRINT }}
```

Comment:
WINDOWS_CERTIFICATE_ZOO_THUMBPRINT can be deleted: certificate thumbprints aren't secrets to begin with, and in this case I need to have it in electron-builder.yml config file, and don't really want to go into the trouble of hot replacing it